src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,145 @@
+package calculator;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Stack;
+import java.util.StringTokenizer;
+
+class Node {
+    String value;
+    Node left;
+    Node right;
+
+    Node(String v) {
+        value = v;
+        left = right = null;
+    }
+}
+public class Calculator {
+
+    private Integer result = null;
+    private Node _root = null;
+    private String _infix = null;
+    private List<String> _postfix = null;
+
+    private boolean isOperator(String c) {
+        return "+".equals(c) || "-".equals(c) || "*".equals(c)|| "/".equals(c);
+    }
+
+
+    public void constructTree() {
+        if(_root == null) {
+            if(_postfix == null) {
+                _postfix = toPostfix(_infix);
+            }
+            Stack<Node> st = new Stack<>();
+            Node t, t1, t2;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/app/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4510206)